dbt/include/spark/profile_template.yml

```diff
@@ -0,0 +1,43 @@
+fixed:
+  type: spark
+prompts:
+  host:
+    hint: [yourorg.sparkhost.com]
+  _choose_authentication_method:
+    odbc:
+      _fixed_method: odbc
+      driver:
+        hint: 'path/to/driver'  
+        schema: 
```

Comment:
Does this need a hint? Not clear to me what this is, but then I'm not spark competent

Reply:
Actually is this line just a typo?
```suggestion
```